readme.md

```diff
@@ -183,7 +183,7 @@
 - [RxJS](https://github.com/reactivex/rxjs) - Functional reactive library for transforming, composing, and querying various kinds of data.
 - [Lazy.js](https://github.com/dtao/lazy.js) - Utility library similar to lodash/Underscore but with lazy evaluation, which can translate to superior performance in many cases.
 - [Kefir.js](https://github.com/kefirjs/kefir) - Reactive library with focus on high performance and low memory usage.
-
+- [Segal Decorators](https://github.com/danrevah/segal-decorators) - Bunch of highly useful decorators, helps in writing a more concise code while improving readability.
```

Comment:
I am just suggesting a change that I feel adheres better to these two requirements:
- Keep descriptions short and simple, but descriptive.
- Check your spelling and grammar.

Perhaps you could change the description to:
`Collection of highly useful decorators that help write concise code while also improving readability.`

Reply:
Ack.

Changed.